Documentation/Books/Manual/SUMMARY.md

```diff
@@ -130,36 +130,45 @@
   * [Storage Engines](Architecture/StorageEngines.md)
   * [Replication](Architecture/Replication/README.md)
   * [Write-ahead log](Architecture/WriteAheadLog.md)
-* [Foxx Microservices](Foxx/README.md)
+* [Foxx services](Foxx/README.md)
```

Comment:
@pluma Is this approved by management/marketing?

Reply:
Good point, I'll revert that. I initially changed this because most of the docs just use "services" but I guess for the chapter title we should stick with it unless there's a deliberate decision. Thanks for catching this.